test/helpers/react_components/mentoring/session_test.rb

```diff
@@ -52,9 +52,10 @@ class SessionTest < ReactComponentTestCase
             avatar_url: student.avatar_url,
             reputation: student.formatted_reputation,
             is_favorite: false,
-            num_previous_sessions: 15,
+            num_previous_sessions: 0,
```

Comment:
Could you add a previous session, so that we can check this works pls?

Reply:
Done.